examples/remove-incomplete-upload.mjs

```diff
@@ -0,0 +1,35 @@
+/*
+ * MinIO Javascript Library for Amazon S3 Compatible Cloud Storage, (C) 2024 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Note: YOUR-ACCESSKEYID, YOUR-SECRETACCESSKEY and my-bucketname
+// are dummy values, please replace them with original values.
```

Comment:
Thanks for the change @prakashsvmx, however it looks like we don't have `my-bucketname` in this file and we are missing `my-objectname` which is mentioned, can we make the following change?

```suggestion
// Note: YOUR-ACCESSKEYID, YOUR-SECRETACCESSKEY, test-bucket and
// my-objectname are dummy values, please replace them with original values.
```

Reply:
my bad. thank you , updated.